geniza/common/metadata_export.py

```diff
@@ -165,3 +166,32 @@ def http_export_data_csv(self, fn=None):
         response = StreamingHttpResponse(iterr, content_type="text/csv; charset=utf-8")
         response["Content-Disposition"] = f"attachment; filename={fn}"
         return response
+
+
+class LogEntryExporter(Exporter):
+    model = LogEntry
+    csv_fields = [
+        "action_time",
+        "user",
+        "content_type",
+        "content_type_app",
+        "object_id",
+        "change_message",
+        "action",
+    ]
+
+    def get_queryset(self):
+        return super().get_queryset().select_related("content_type", "user")
+
+    action_label = {ADDITION: "addition", CHANGE: "change", DELETION: "deletion"}
```

Comment:
Suppose we could move this up a few lines to keep it consistent with other attributes

Reply:
good point, will do